### PR TITLE
fix(FOROME-683): show locus in title from recieved hg mode

### PR DESCRIPTION
**Pull request by ghost**
https://quantori.atlassian.net/browse/FOROME-683

### CODE REVIEW

**Review by AlMaQntr — APPROVED** [review]

File: src/components/variant/ui/header.tsx [hunk 1]
```diff
@@ -23,9 +28,25 @@ export const VariantHeader = observer(({ setLayout }: Props): ReactElement => {
...
+
+  const genesRow = rows.find(element => element?.name === 'genes')
+  const variantWithoutGenesName = 'None'
+  const genes = genesRow?.cells[0][0] ?? variantWithoutGenesName
+
+  const hg19Row = rows.find(element => element.name === 'hg19')
+  const hg19locus = hg19Row?.cells[0][0] ?? ''
+
+  const hg38Row = rows.find(element => element.name === 'hg38')
+  const hg38locus = hg38Row?.cells[0][0] ?? ''
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/header.tsx]: `const findLocusInRow = (rowName) => {..}`
And move it outside the component function

File: src/components/variant/ui/header.tsx [hunk 2]
```diff
@@ -23,9 +28,25 @@ export const VariantHeader = observer(({ setLayout }: Props): ReactElement => {
   const history = useHistory()
   const location = useLocation()
 
-  const genInfo = get(toJS(variantStore.variant), '[0].rows[0].cells[0][0]', '')
+  const variant = toJS(variantStore.variant)
+  const rows: IAttributeDescriptors[] = get(variant, '[0].rows', [])
+
+  const genesRow = rows.find(element => element?.name === 'genes')
+  const variantWithoutGenesName = 'None'
+  const genes = genesRow?.cells[0][0] ?? variantWithoutGenesName
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/header.tsx]: `const findGenesInRow = () => {..}`
And move it outside the component function

File: src/components/variant/ui/header.tsx [hunk 3]
```diff
@@ -23,9 +28,25 @@ export const VariantHeader = observer(({ setLayout }: Props): ReactElement => {
...
+  const hg19locus = hg19Row?.cells[0][0] ?? ''
+
+  const hg38Row = rows.find(element => element.name === 'hg38')
+  const hg38locus = hg38Row?.cells[0][0] ?? ''
+
+  // TODO: update type after implantion IDsInfo interface
+  const { meta }: any = toJS(dirinfoStore.dsinfo)
+  const hgModeValue: HgModes = meta?.modes?.[0]
+
+  const currentLocus = hgModeValue === HgModes.HG19 ? hg19locus : hg38locus
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/header.tsx]: maybe `currentLocus` could be a getter in `dirinfoStore`? I don't insist but it's a subject to think about

File: src/components/variant/ui/header.tsx (hunk 2 above)
Comment (ghost) [reply src/components/variant/ui/header.tsx]: Moved

File: src/components/variant/ui/header.tsx (hunk 3 above)
Comment (ghost) [reply src/components/variant/ui/header.tsx]: Fixed

File: src/components/variant/ui/header.tsx (hunk 1 above)
Comment (ghost) [reply src/components/variant/ui/header.tsx]: Moved

**Review by Lapk1n — APPROVED** [review]

**Pull request merged**